Cargo.toml

```
@@ -27,8 +27,8 @@ base64 = "0.21.7"
 borsh = { version = "1.3.0" , features = ["derive", "de_strict_order"]}
 futures = "0.3.30"
 hex = "0.4.3"
-ibc-proto = { version = "0.51.1", default-features = false, features = ["serde"], optional = true }
-ibc-types = { version = "0.15.0", default-features = false, features = ["std"] }
+ibc-proto = { version = "0.52", default-features = false, features = ["serde"], optional = true }
```

I am a fan of providing full versions, including the minor/patch part:
```suggestion
ibc-proto = { version = "0.52.0", default-features = false, features = ["serde"], optional = true }
```